go.mod

```diff
@@ -1,25 +1,25 @@
 module sigs.k8s.io/cluster-api-operator
 
-go 1.17
+go 1.19
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.2.0
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.3.0
 
 require (
-	github.com/google/go-cmp v0.5.8
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.19.0
+	github.com/google/go-cmp v0.5.9
+	github.com/onsi/ginkgo/v2 v2.5.1
```

Comment:
Maybe bump to 2.6.1 while we are here?

Reply:
I bumped it to 2.7.0, as it it the latest released version.